packages/server/src/admin/client.test.ts

```diff
@@ -1,9 +1,14 @@
 import { randomUUID } from 'crypto';
 import express from 'express';
 import request from 'supertest';
+import { getReferenceString, resolveId } from '@medplum/core';
+
```

Comment:
Remove blank lines & "Organize imports"

Reply:
Yeah... this is a gap in our ESLint config right now.  Currently we're just using the VS Code default behavior.  It would be nice to enforce with ESLint/Prettier.